docs/development.md

```diff
@@ -33,9 +33,6 @@ your `~/.npmrc` contains as `prefix` some location you can write.
 
 Start development mode:
 
-    cd adhocracy4
-    npm link
-
     cd ../$PROJECT
     pip install -e ../adhocracy4/
     npm link adhocracy4
```

Comment:
this needs to be the relative path of a4 like line 37 above.

Reply:
@hom3mad3 line 36 + 37 should be removed like you did, but as @m4ra says `npm link adhocracy4` should either become `npm link <path to your adhocracy4 folder>` or `npm link ../adhocracy4` (assuming that everyone cloned a4 in  the parent folder)